tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -63,4 +73,28 @@ public Response headZepplin() {
     public Response getZepplin() {
         return Response.ok().build();
     }
+
+    @POST
+    @APIResponse(responseCode = "200", description = "Review posted")
+    @APIResponse(responseCode = "404", description = "Review not found")
+    @Operation(summary = "Post by Zepplin", operationId = "postZepplin")
+    @Produces("text/plain")
+    @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
+    @Consumes("multipart/form-data")
+    public Response postZepplin(
+            @RequestBody(description = "Record of a new user to be created in the system.", required = true,
+                         content = @Content(mediaType = "application/json",
```

Comment:
I think this should match the `@Consumes` type. Of course consumes could have multiple entries and there could be multiple `@Content` annotations, but for the purposes of the `encoding` they should match.
```suggestion
                         content = @Content(mediaType = "multipart/form-data",
```

Reply:
All done and tested.